core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/ViewNames.java

```diff
@@ -131,10 +118,7 @@ public static ViewTypes getViewType(String viewName) {
                 return ViewTypes.TOKEN_LABEL_VIEW;
             case SENTENCE:
             case PARAGRAPH:
-            case NER_ACE_COARSE_HEAD:
-            case NER_ACE_FINE_HEAD:
-            case NER_ACE_COARSE_EXTENT:
-            case NER_ACE_FINE_EXTENT:
+            case NER_ACE:
```

Comment:
I think for clarity we should call this "MENTION_ACE" as it has non-Named Entity mentions (correct me if I'm wrong)